hw1/02-palindrome.html

```diff
@@ -6,22 +6,29 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
     <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
+      <section
+        class="bg-light p-3 mx-auto mt-5 w-50"
+        style="padding-left: 25px"
+      >
         <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
+          <h1>Exercise 02 - Palindrome</h1>
+          <div class="mt-4 mb-4">
+            <label>Enter a positive number:</label>
```

Comment:
This label tag and the following input tag as they are will trigger a Accessibility error. your label needs the "for" attribute set to the ID value of the input tag (the input tag needs a ID attribute.